### ISSUE TITLE
[bitnami/cassandra]: ENABLE SASI INDEXES

### ISSUE THREAD

**Issue by tancho-mihov-1crew** [bitnami/charts#17467]
### Name and Version

bitnami/cassandra 10.4.2

### What architecture are you using?

amd64

### What steps will reproduce the bug?

The issue is same as issue [CASSANDRA: ENABLE SASI INDEXES](https://github.com/bitnami/charts/issues/8634)
I want to set `sasi_indexes_enabled: true`
When I'm using the suggestion "to be used a configmap with your custom configuration cassandra.yaml" it works **only when** `replicaCount: 1`
I want to use 3 replicas:
`replicaCount: 3`
Because 
Usage of `existingConfiguration` overrides any other Cassandra configuration set in the chart.
Properties like `listen_address` and `broadcast_rpc_address` expected to be uncial per replica but they are the same as values from the configmap. And it makes any other instances that first one to not run.

### What do you see instead?

Any other instances that first one to not run

**Comment by rafariossaa** [comment]
Hi, 
  When you said `it only works when replicaCount: 1` you mean you are getting errors ?

**Comment by tancho-mihov-1crew** [comment]
Hi,
When `replicaCount: 3` as desired first container cassandra-0 runs as expected, but the second one:
```
  Warning  Unhealthy  14m (x12379 over 11d)    kubelet  Liveness probe failed: nodetool: Failed to connect to '127.0.0.1:7199' - ConnectException: 'Connection refused (Connection refused)'.
  Warning  BackOff    9m45s (x30884 over 11d)  kubelet  Back-off restarting failed container cassandra in pod cassandra-1_non-production(1f7ffdca-3c83-4ba6-a9d0-fc7e6179021a)
  Warning  Unhealthy  4m43s (x51536 over 11d)  kubelet  Readiness probe failed: nodetool: Failed to connect to '127.0.0.1:7199' - ConnectException: 'Connection refused (Connection refused)'.
```
because `cassandra.yaml` is "hard coded" by setting `existingConfiguration` and second and third containers gets exact the same cassandra.yaml` but they expected personal values for `listen_address` and `broadcast_rpc_address`.

**Comment by rafariossaa** [comment]
Hi,
  As you indicated currently this way of passing the configuration is not supported if you have more than one replica.
  I am opening an internal task to study this further and try to look for a fix/workaround. I can not provide an ETA for this, meanwhile maybe you would like to send a PR to fix this. We will be glad to review and merge it.

**Comment by trnikon** [comment]
Setting changes to cassandra.yaml is essential, and I am having trouble doing the same. I would say most production environments use cassandra in cluster mode.

**Comment by rafariossaa** [comment]
Hi,
  Thanks for your feedback

**Comment by vamayufei** [comment]
Thanks for maintaining the charts, team 🥇. Unfortunately, not being able to use custom `cassandra.yaml` in cluster mode is a blocker for me as well.

**Comment by NikulinIgor** [comment]
@tancho-mihov-1crew Hi! I found the solution, use that in values.yaml:

extraEnvVars:
  - name: CASSANDRA_CFG_YAML_SASI_INDEXES_ENABLED
    value: "true"

All env's with names "CASSANDRA_CFG_YAML_*" startup scripts put in cassandra.yaml

**Comment by NikulinIgor** [comment]
@rafariossaa fyi

**Comment by jotamartos** [comment]
>All env's with names "CASSANDRA_CFG_YAML_*" startup scripts put in cassandra.yaml

That's correct, "CASSANDRA_CFG_YAML_*" will add the corresponding key and the provided value to cassandra.yaml as mentioned in the [Bitnami Cassandra's README](https://github.com/bitnami/containers/tree/main/bitnami/cassandra).

Closing this ticket now. Thanks
